class Trycatch
{
	public static void main(String ar[])
	{

		System.out.println("Hello");
		try
		{
		int a=10/0;
		}
		catch(ArithmeticException b)
		{
			System.out.println("Your ArithmeticException Arrare");

		}
		
	
		

	}

}